docs/arcadia-users-group/20240416-intro-to-ml1/make_peptides_ml_dataset.md

```diff
@@ -0,0 +1,263 @@
+
+Overall Statistics
+
+               Accuracy : 0.9849
+                 95% CI : (0.9705, 0.9935)
+    No Information Rate : 0.4396
+    P-Value [Acc > NIR] : < 2.2e-16
+
+                  Kappa : 0.98
+
+ Mcnemar's Test P-Value : NA
+
+Statistics by Class:
+
+                     Class: antimicrobial Class: coagulation_or_anticoagulation Class: growth_factor Class: hormone Class: immunological_activity Class: metabolic Class: neurological_activity Class: propeptide Class: signal_peptide Class: therapeutic
+Sensitivity                        1.0000                               1.00000              0.68421        1.00000                        0.9839          1.00000                      1.00000            1.0000                0.9957                1.0
+Specificity                        1.0000                               0.99606              1.00000        1.00000                        1.0000          1.00000                      0.99806            1.0000                0.9832                1.0
+Pos Pred Value                     1.0000                               0.92000              1.00000        1.00000                        1.0000          1.00000                      0.93333            1.0000                0.9789                1.0
+Neg Pred Value                     1.0000                               1.00000              0.98839        1.00000                        0.9979          1.00000                      1.00000            1.0000                0.9966                1.0
+Prevalence                         0.1264                               0.04340              0.03585        0.03396                        0.1170          0.02075                      0.02642            0.0566                0.4396                0.1
+Detection Rate                     0.1264                               0.04340              0.02453        0.03396                        0.1151          0.02075                      0.02642            0.0566                0.4377                0.1
+Detection Prevalence               0.1264                               0.04717              0.02453        0.03396                        0.1151          0.02075                      0.02830            0.0566                0.4472                0.1
+Balanced Accuracy                  1.0000                               0.99803              0.84211        1.00000                        0.9919          1.00000                      0.99903            1.0000                0.9894                1.0
+```
+
+## Documentation of data set contents
+
+### Column descriptions
+
+* `rep`: representative peptide id for the cluster. Generated by running MMSeqs2 `linclust` on the peptipedia database with an identity threshold of 30%.
+* `member`: cluster member peptide id.
+* `member_sequence`: cluster member peptide sequence.
```

Comment:
Could we use more descriptive names for these columns? Also, I think we could drop the `rep` column in the final dataset, right? (it might otherwise be confusing)